src/Authoring/WinRT.SourceGenerator/Generator.cs

```diff
@@ -271,13 +271,16 @@ public static int DllGetActivationFactory(void* activatableClassId, void** facto
 
                 """);
 
-            // There's three possible cases for this method:
+            // There's four possible cases for this method:

```

Comment:
Should we add a C# exe project to reference the authoring project and set this property to exercise these changes in the build?